website/app.py

```diff
@@ -119,6 +123,8 @@ def init_app(settings_module='website.settings', set_backends=True, routes=True,
     init_addons(settings, routes)
     build_js_config_files(settings)
 
+    django.setup()
```

Comment:
It is there to make reverse work. I'm trying to find an alternative because this is breaking admin. We can't have django settings pointing at api to make reverse work and admin to make admin work at the same time.